src/abstractdataframe/abstractdataframe.jl

```diff
@@ -558,14 +558,19 @@ Base.first(df::AbstractDataFrame) = df[1, :]
     first(df::AbstractDataFrame, n::Integer; view::Bool=false)
 
 Get a data frame with the `n` first rows of `df`.
+If `n` is greater than number of rows of `df` get all rows.
```

Comment:
For consistency with other sentences:
```suggestion
Get all rows if `n` is greater than the number of rows in `df`.
```